README.md

```diff
@@ -4,6 +4,40 @@ A lightweight IPNI node mapping Filecoin PieceCID → payload block CID.
 
 - [Design doc](./docs/design.md)
 
+## Basic use
+
+Note: this will change soon, see
+https://github.com/filecoin-station/piece-indexer/issues/33
+
+## `GET /sample/{provider-id}/{piece-cid}
+
+Example:
+
+https://pix.filspark.com/sample/12D3KooWHKeaNCnYByQUMS2n5PAZ1KZ9xKXqsb4bhpxVJ6bBJg5V/baga6ea4seaqlwzed5tgjtyhrugjziutzthx2wrympvsuqhfngwdwqzvosuchmja
+
+```json
+{
+  "samples": ["bafkreigrnnl64xuevvkhknbhrcqzbdvvmqnchp7ae2a4ulninsjoc5svoq"]
+}
```

Comment:
I see it returns samples, but what is a sample?

Reply:
Good call!

https://github.com/ipni/xedni/blob/526f90f5a6001cb50b52e6376f8877163f8018af/openapi.yaml#L11
> Sample a set of multihashes ingested by IPNI for a given ContextID

Let me add some explanation to README.